libs/domain/site/dropdown-item/stories/demo.stories.ts

```diff
@@ -0,0 +1,36 @@
+import { getAppIcons } from '@/tools/storybook';
+import { Meta, Story } from '@storybook/web-components';
+import { TemplateResult, html } from 'lit';
+import { storybookPrefix } from '../../.constants';
+import {
+  DropdownItemContent,
+  DropdownItemOptions,
+} from '../dropdown-item.model';
+
+const icons = getAppIcons();
```

Comment:
component has hover state, better to ignore this story for chromatic and add static one, with default and hovered state